README.md

```diff
@@ -159,6 +159,42 @@ docker-compose -f docker/app.yml up
 
 In case of Oracle database, see [official documentation](https://github.com/oracle/docker-images/blob/master/OracleDatabase/SingleInstance/README.md)
 
+
+## Using database migrations
+
+If you had alredy run the application before creating your first migration, some database tables might be alredy 
```

Comment:
Already

Reply:
Sorry, actually english is not my main language. I will be correcting it soon.

Reply:
no problem =)